expression/builtin_string_vec.go

```diff
@@ -404,11 +405,37 @@ func (b *builtinSubstringIndexSig) vecEvalString(input *chunk.Chunk, result *chu
 }
 
 func (b *builtinUnHexSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinUnHexSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+
+	result.ReserveString(n)
+	for i := 0; i < n; i++ {
+		if buf.IsNull(i) {
+			result.AppendNull()
+			continue
+		}
+		str := buf.GetString(i)
+		if len(str)%2 != 0 {
+			str = "0" + str
+		}
+		bs, e := hex.DecodeString(str)
+		if e != nil {
+			return e
```

Comment:
According to the row-based implementation, we should set it to null and ignore this error instead of throwing it.

Reply:
OK